source/more-info/unsupported/apparmor.markdown

```diff
@@ -10,7 +10,7 @@ without this, the Supervisor is missing important security mechanics to protect
 
 ## The solution
 
-If the AppArmor is not enabled on your host, set this to the Linux boot cmd: `apparmor=1 security=apparmor` and reboot your operating system.
+If the AppArmor is not enabled on your host, add this to the Linux kernel boot parameters:  `apparmor=1 security=apparmor` and then reboot your operating system.
```

Comment:
It the same wording?

https://www.kernel.org/doc/html/v4.14/admin-guide/kernel-parameters.html

Reply:
* In my opinion, 'add' is more descriptive about what needs to be done.
* Note that the page you referenced is called kernel "parameters."  From what I see, that's the most common word used, including being used by Debian: https://www.debian.org/releases/stable/amd64/ch05s03.en.html